src/components/app/SingleStrategy/index.tsx

```diff
@@ -16,24 +16,23 @@ import StrategyDetail from './StrategyDetail';
 import StrategyHealthCheck from './StrategyHealthCheck';
 
 import { ErrorAlert } from '../../common/Alerts';
-import { Strategy, Vault, Network, DEFAULT_NETWORK } from '../../../types';
+import { Network, DEFAULT_NETWORK } from '../../../types';
 
 import BreadCrumbs from './BreadCrumbs';
 import EtherScanLink from '../../common/EtherScanLink';
 import ReactHelmet from '../../common/ReactHelmet';
 import ProgressSpinnerBar from '../../common/ProgressSpinnerBar/ProgressSpinnerBar';
 
-import { getService as getVaultService } from '../../../services/VaultService';
-import {
-    getError,
-    getReportsForStrategies,
-    getStrategies,
-    getWarnings,
-} from '../../../utils';
+import { getError } from '../../../utils/error';
+import { getWarnings } from '../../../utils/warnings';
+import { getReportsForStrategies } from '../../../utils/reports';
 
 import StrategyReports from './StrategyReports';
 import { GlobalStylesLoading } from '../../theme/globalStyles';
 import { useStrategyReportContext } from '../../../contexts/StrategyReportContext';
+import useVault from '../../../hooks/use-vault';
```

Comment:
Maybe we could create an `index.js` in the hooks folder and import these all from the same location as well? `import { useVault, useStrategy, useStrategyMetadata } from '../../../hooks';`